Content.Server/Kitchen/Components/ReagentGrinderComponent.cs

```diff
@@ -31,6 +31,9 @@ public sealed partial class ReagentGrinderComponent : Component
         [DataField("juiceSound"), ViewVariables(VVAccess.ReadWrite)]
         public SoundSpecifier JuiceSound { get; set; } = new SoundPathSpecifier("/Audio/Machines/juicer.ogg");
 
+        [DataField("autoMode"), ViewVariables(VVAccess.ReadWrite)]
+        public GrinderAutoMode AutoMode = GrinderAutoMode.Off;
+
```

Comment:
Datafield strings are obsoloete my dude, and VVRW is redundant here as that's the default.

Reply:
Fixed